rustls/src/msgs/deframer.rs

```diff
@@ -338,6 +328,13 @@ struct DeframerVecBuffer {
 }
 
 impl DeframerVecBuffer {
+    /// Returns true if we have messages for the caller
+    /// to process, either whole messages in our output
+    /// queue or partial messages in our buffer.
+    pub fn has_pending(&self) -> bool {
```

Comment:
> This should drift to the bottom of the impl block. 

shouldn't this stay at the top as per [the style guide](https://github.com/rustls/rustls/blob/078f03334b2de9d8e464755d8836318b9a221346/CONTRIBUTING.md#ordering-associated-functions-within-an-inherent-impl-block) because it's public API?

> Please use the full 100-char width for doc strings. Ideally the first line stands on it own as a concise description.

ok but note that this an existing doc string that I have simply moved around.

Reply:
> shouldn't this stay at the top as per [the style guide](https://github.com/rustls/rustls/blob/078f03334b2de9d8e464755d8836318b9a221346/CONTRIBUTING.md#ordering-associated-functions-within-an-inherent-impl-block) because it's public API?

Err, yes -- sorry for the noise.

> > Please use the full 100-char width for doc strings. Ideally the first line stands on it own as a concise description.
> 
> ok but note that this an existing doc string that I have simply moved around.

Okay, that would have been more obvious if it wasn't part of a large commit with a bunch of other stuff.